tools/crm_simulate.c

```diff
@@ -989,7 +997,7 @@ main(int argc, char **argv)
     data_set = pe_new_working_set();
     if (data_set == NULL) {
         crm_perror(LOG_ERR, "Could not allocate working set");
-        rc = -ENOMEM;
+        rc = ENOMEM;
```

Comment:
Something that would be good to take care of now would be to ditch the use of crm_perror() in crm_simulate.

I eventually want to get rid of crm_perror() entirely. Unlike all the other logging functions, it both logs and prints to stderr. While that's confusing enough, it also will cause us headaches with formatted output.

In crm_simulate, we can set the GError instead. This message doesn't need to go to the log (which in crm_simulate's case means verbose output).

Reply:
Good idea.  I've added this into 26973dd (Fix: tools: Use GError in crm_simulate where possible.).  Let me know what you think.